crates/miden-objects/src/note/note_tag.rs

```diff
@@ -28,6 +29,11 @@ const LOCAL_PUBLIC_ANY: u32 = 0x8000_0000;
 // The 2 most significant bits are set to `0b11`.
 const LOCAL_ANY: u32 = 0xc000_0000;
 
+// Creating a tag for local use case directly from AccountId defaults to using 14 bits.
+pub const DEFAULT_LOCAL_TAG_LENGTH: u8 = 14;
+/// The maximum number of bits that can be encoded into the tag for local accounts.
+pub const MAX_LOCAL_TAG_LENGTH: u8 = 30;
+
```

Comment:
not sure where these should live - it feels weird that `Address` imports them.
Maybe better in `miden-objects/src/lib.rs`?